test/server/server_test.cc

```diff
@@ -32,7 +33,6 @@ using testing::StrictMock;
 
 namespace Envoy {
 namespace Server {
-namespace {
```

Comment:
This removal is needed, without this the actual friend class is not working - probably if we want to retain this we may have qualify including anonymous namespace while declaring it as friend. I thought it is OK to remove this.

Reply:
I think you can revert this change now.